### ISSUE TITLE
Mishandling of trailing comments

### ISSUE THREAD

**Issue by Conduitry** [Rich-Harris/code-red#28]
Trailing comments have some buggy handling, which _most_ of the time just results in them being attached to the wrong line in the generated code, which isn't a huge deal, but there are also cases where they can result in syntactically incorrect output.

```js
const cr = require('code-red');
const foo = cr.b`function foo() {
// hey1
/*
hey2
*/
}`;
```

->

```
[
  Node {
    type: 'FunctionDeclaration',
    start: 0,
    end: 37,
    id: Node { type: 'Identifier', start: 9, end: 12, name: 'foo' },
    expression: false,
    generator: false,
    async: false,
    params: [],
    body: Node {
      type: 'BlockStatement',
      start: 15,
      end: 37,
      body: [],
      trailingComments: [ { type: 'Line', value: ' hey1', start: 17, end: 24 } ]
    },
    trailingComments: [ { type: 'Block', value: '\nhey2\n', start: 25, end: 35 } ]
  }
]
```

```js
console.log(cr.print(foo).code);
```

->

```
function foo() {

} // hey1 /*
hey2
*/
```

The comments end up outside the function block, but more seriously, the block comment ends up starting on the same line as the line comment.

Ref. https://github.com/sveltejs/svelte/issues/4312

**Comment by vipero07** [comment]
Would it be a bad idea for the compiler to ignore comments? Considering minification results in removal of the comments.

**Comment by Conduitry** [comment]
The comments can be important - partial support for preserving them was recently added for https://github.com/sveltejs/svelte/issues/4268

**Comment by Rich-Harris** [comment]
Fixed in 0.0.32
